src/index.js

```diff
@@ -313,7 +313,7 @@ const stacLayer = async (data, options = {}) => {
             url: href,
             asset,
             key,
-            item: asset,
+            item: data,
```

Comment:
@m-mohr thanks for catching this issue!  I totally missed it.  I'm a little concerned that if we use `"item"` to represent different types (asset, item or collection), it might make things more ambiguous.  Would it be possible to add a collection and data parameter?  So it would look like this:
```js
const layer = await stacLayer(data, {
  buildTileUrlTemplate: ({
    data, // the original STAC data (collection, item or asset)
    asset, // the STAC Asset object
    item, // the STAC item / feature
    collection, // the STAC collection if available
    // href, bounds, key, isCOG, and isVisual stay the same
  }) => {
     // 
  }
});```

Reply:
fwiw, I'm sorry I overlooked this.  I shouldn't have been passing in asset as "item".  At the time it made sense to me, but I don't think I agree now.

Reply:
Yes, although we could also call it just "stac" or "context" and then it's a single place where you get it from. Having two inputs for item and collection makes it a bit more verbose downstream (at least in the use case I'm looking at right now).

Reply:
got it. `stac` works for me!

Reply:
Okay, I renamed it in the last commit.